### PR TITLE
Switch to docker

### PR DESCRIPTION
**Pull request by gfaivre**

### CODE REVIEW

**Comment by nervo** [comment]
Still wip ?

**Comment by gfaivre** [comment]
Nop cf "labels" field ;)

File: Makefile [hunk 1]
```diff
@@ -21,60 +21,120 @@ help:
...
-
-crop-headers:
-	find public/fr/images/posts/headers \( -iname "*.png" -o -iname "*.jpg" -o -iname "*.jpeg" \) -type f -exec mogrify -resize 2000x {} \;
-	find public/en/images/posts/headers \( -iname "*.png" -o -iname "*.jpg" -o -iname "*.jpeg" \) -type f -exec mogrify -resize 2000x {} \;
+#######
+# Dev #
+#######
+
+## Dev
+dev:
```
Comment (hyvs) [inline Makefile]: rename to `make shell` or `make bash` ?

File: Makefile [hunk 2]
```diff
@@ -21,60 +21,120 @@ help:
...
-	hugo server --theme=blog --buildDrafts --watch --ignoreCache=true
-
-## Hugo server FR (Dev only)
-server-start-fr:
-	hugo server --theme=blog --buildDrafts --watch --ignoreCache=true --config=config_fr.yaml
-
-## Hugo server EN (Dev only)
-server-start-en:
-	hugo server --theme=blog --buildDrafts --watch --ignoreCache=true --config=config_en.yaml
+build:
```
Comment (hyvs) [inline Makefile]:
```
build: install
```

**Comment by hyvs** [comment]
Minor suggestions, LGTM 👍

**Pull request merged**